### DISCUSSION TITLE
Welcome to django-hurricane Discussions!

### DISCUSSION THREAD

**Discussion by Schille** [Blueshoe/django-hurricane#40] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by scheung38** [comment]
Could someone comment what is the difference between the repos: hurricane-django vs hurricane-based-helm-template?

Which one should we use if needing to deploy Django as well as a database?

Do we clone this repo, hurricane-based-helm-template or both given the context?

**Comment by tschale** [comment]
Hi @scheung38,
the repo [django-hurricane](https://github.com/django-hurricane/django-hurricane) contains the source code of Hurricane. You only need to clone this repo if you want to work on hurricane itself. If you just want to use hurricane, you don't need to clone it.
[hurricane-based-helm-template](https://github.com/django-hurricane/hurricane-based-helm-template) contains a cookiecutter template to generate Helm charts to deploy a Django application that uses Hurricane. The charts don't generate a database, but of course you can add a database there, or install it in parallel.

**Reply by scheung38** [reply]
@tschale thanks is this tricky to extend if we need to have others like Cert-Manager, NGINX,  MongoDB, ElasticSearch, (ELK) etc to be deployed to Kubernetes (AKS to be precise) 

Also what latest Django/Python does it support as I ran it but couldn't even get it to start?
